extensions/pkg/controller/worker/genericactuator/interface.go

```diff
@@ -52,10 +52,23 @@ type WorkerDelegate interface {
 	UpdateMachineImagesStatus(context.Context) error
 
 	// DeployMachineDependencies is a hook to create external machine dependencies.
+	// DEPRECATED: Use PreReconcileHook() instead.
+	// TODO(dkistner) Remove in future release.
```

Comment:
The proper way to mark a func as deprecated is:
> To signal that an identifier should not be used, add a paragraph to its doc comment that begins with "Deprecated:" followed by some information about the deprecation.

Ref http://blog.golang.org/godoc-documenting-go-code

```suggestion
	// Deprecated: Use PreReconcileHook() instead.
	// TODO(dkistner): Remove in a future release.
```